src/CMakeLists.txt

```diff
@@ -360,6 +360,16 @@ if(WIN32)
 	)
 endif(WIN32)
 
+# On apple only
+if(APPLE)
+	set(libwesnoth-game_STAT_SRC
+		${libwesnoth-game_STAT_SRC}
+		desktop/apple_notification.hpp
```

Comment:
I don't think you need to add the hpp file here. I think adding jsut the cpp file should be sufficient.